bigtable/writes/writes_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package writes
+
+import (
+	"bytes"
+	"context"
+	"os"
+	"strings"
+	"testing"
+
+	"cloud.google.com/go/bigtable"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestWrites(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	ctx := context.Background()
+	project := os.Getenv("GOLANG_SAMPLES_BIGTABLE_PROJECT")
+	instance := os.Getenv("GOLANG_SAMPLES_BIGTABLE_INSTANCE")
+	if project == "" || instance == "" {
+		t.Skip("Skipping bigtable integration test. Set GOLANG_SAMPLES_BIGTABLE_PROJECT and GOLANG_SAMPLES_BIGTABLE_INSTANCE.")
+	}
+	adminClient, err := bigtable.NewAdminClient(ctx, project, instance)
+
+	tableName := "mobile-time-series-" + tc.ProjectID
+	adminClient.DeleteTable(ctx, tableName)
+
+	if err := adminClient.CreateTable(ctx, tableName); err != nil {
+		t.Fatalf("Could not create table %s: %v", tableName, err)
+	}
+
+	columnFamilyName := "stats_summary"
+	if err := adminClient.CreateColumnFamily(ctx, tableName, columnFamilyName); err != nil {
+		t.Fatalf("CreateColumnFamily(%s): %v", columnFamilyName, err)
+	}
+
+	buf := new(bytes.Buffer)
+	if err = writeSimple(buf, project, instance, tableName); err != nil {
+		t.Errorf("TestWriteSimple: %v", err)
+	}
+
+	got := buf.String()
+	if want := "Successfully wrote row"; !strings.Contains(got, want) {
+		t.Errorf("got %q, want %q", got, want)
+	}
+
+	buf.Reset()
+	if err = writeConditionally(buf, project, instance, tableName); err != nil {
+		t.Errorf("TestWriteConditionally: %v", err)
+	}
+
+	got = buf.String()
+	if want := "Successfully updated row's os_name"; !strings.Contains(got, want) {
+		t.Errorf("got %q, want %q", got, want)
+	}
+
+	buf.Reset()
+	if err = writeIncrement(buf, project, instance, tableName); err != nil {
+		t.Errorf("TestWriteIncrement: %v", err)
+	}
+
+	got = buf.String()
+	if want := "Successfully updated row"; !strings.Contains(got, want) {
+		t.Errorf("got %q, want %q", got, want)
+	}
+
+	buf.Reset()
+	if err = writeBatch(buf, project, instance, tableName); err != nil {
+		t.Errorf("TestWriteBatch: %v", err)
+	}
+
+	got = buf.String()
+	if want := "Successfully wrote 2 rows"; !strings.Contains(got, want) {
```

Comment:
Might not want to include "2" here in case the sample ever changes.

Reply:
If the samples are changing, the tests should be changed too.